dotcom-rendering/src/web/components/CommercialMetrics.tsx

```diff
@@ -16,7 +17,16 @@ export const CommercialMetrics: React.FC<{
 		false,
 	);
 
+	const [adBlockerInUse, setAdBlockerInUse] = useState<boolean>();
+
+	isAdBlockInUse().then((value) => {
+		setAdBlockerInUse(value);
+	}).catch((error) => {
+		console.log(error);
+	});
+
 	useEffect(() => {
```

Comment:
This is very much outside this PR but this logic could be greatly simplified using `useOnce`. This type of use case where you only want to send a metric one time but only when a series of dependencies are defined (or just ready) is pretty much what that function was built for 😻